config/database.js

```diff
@@ -1,17 +1,17 @@
+// mongoose connection
+const mongoose = require('mongoose');
 
-// const mongoose = require('mongoose');
+mongoose.connect(process.env.DATABASE_URL, {
+    useNewUrlParser: true, 
+    useCreateIndex: true,
+    useUnifiedTopology: true 
+});
 
-// mongoose.connect(process.env.DATABASE_URL, {
-//     useNewUrlParser: true, 
-//     useCreateIndex: true,
-//     useUnifiedTopology: true 
-// });
+const db = mongoose.connection
+db.once('open', _ => {
+  console.log('Database connected:', process.env.DATABASE_URL)
+})
 
-// const db = mongoose.connection
-// db.once('open', _ => {
-//   console.log('Database connected:', process.env.DATABASE_URL)
-// })
-
-// db.on('error', err => {
-//   console.error('connection error:', err)
-// })
+db.on('error', err => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
db.on('error', (err) => {
```